cmd/agent/main.go

```diff
@@ -1,3 +1,50 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	conf "github.com/xoxloviwan/go-monitor/internal/config_agent"
+	metrs "github.com/xoxloviwan/go-monitor/internal/metrics"
+)
+
+var (
+	PollCount int64 = 0
+)
+
+func send(adr *string, urls *[]string) (err error) {
```

Comment:
Массивы и так передаются в функцию по ссылке, а не по значению поэтому указатель тут не нужен.
```suggestion
func send(adr *string, urls []string) (err error) {
```